### ISSUE TITLE
Responsiveness

### ISSUE THREAD

**Issue by chasallen** [meovv/web-iv-portfolio-site#2]
I didn't see a problem with your responsiveness. Nothing gets in your way or causes compositional issues. The fabicon (???I'm guessing that's what the image in the sticky nav is???) seems like it might seem like it's getting in the way on the phone screen size. Maybe not a huge issue though. Advanced.

**Comment by elladurham** [comment]
Yeah, the positioning of the icon on medium screen seems too far out from the corner of the browser to read as a menu. Works much better on mobile.

**Comment by elladurham** [comment]
There's something in your footer structure that is making your positioning of your copyright seem unintentionally in a new row and it doesn't seem to match the wireframes I have on hand either.
